DealerHub.Domain/Dealer.cs

```diff
@@ -27,3 +27,36 @@ private void UpdateTelephone(string newTelephone)
         Telephone = newTelephone;
     }
 }
+
+public class Customer
+{
+    public string FirstName { get; private set; } = string.Empty;
+    public string LastName { get; private set; } = string.Empty;
+    public string Address { get; private set; } = string.Empty;
```

Comment:
Address should be 4 separate fields. AddresLine1, AddressLine2, AddressLine3 and PostCode.